src/main/java/net/dv8tion/jda/core/entities/EntityBuilder.java

```diff
@@ -86,6 +86,11 @@ public SelfUser createSelfUser(JSONObject self)
                 .setBot(self.has("bot") && self.getBoolean("bot"));
     }
 
+    public Game createGameInternal(String name, String url, Game.GameType type)
```

Comment:
This does not need `Internal` appended to it.
It is implied that anything being built by the EntityBuilder is being done by an internal JDA system.